multiqc/modules/sourmash/gather.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse similarity matrix output by sourmash gather """
+
+import csv
+import logging
+from collections import OrderedDict
+
+from multiqc.plots import bargraph
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+
+class GatherMixin:
+    def parse_gather(self):
+        """
+        Modeled after the kraken module and the sourmash compare module.
+        """
+
+        self.top_n = 5
+
+        # find and load gather reports
+        self.gather_raw_data = dict()
+        for f in self.find_log_files("sourmash/gather", filehandles=True):
+            data = []
+            for line in csv.DictReader(f["f"]):
+                data.append(
+                    {
+                        "query_name": line["query_name"],
+                        "match_name": line["name"],
+                        "pct_unique_weighted": float(line["f_unique_weighted"]) * 100,
+                        "pct_match": float(line["f_match"]) * 100,
```

Comment:
It's good to force floats with `100.0`
```suggestion
                        "pct_unique_weighted": float(line["f_unique_weighted"]) * 100.0,
                        "pct_match": float(line["f_match"]) * 100.0,
```

Reply:
But it doesn't make any difference, since the type of the output depends on the type of the first operand?